dialog/src/main/java/roboy/dialog/states/gameStates/GamingTwentyQuestionsState.java

```diff
@@ -43,18 +44,39 @@
 	private boolean guessesAvailable = false;
 	private boolean gameFinished = false;
 	private boolean stopGame = false;
+	private boolean network = false;
 
 	private boolean filterApplied = false;
 	private boolean emotionShown = false;
 	private String winner = "";
 
 	public GamingTwentyQuestionsState(String stateIdentifier, StateParameters params) {
 		super(stateIdentifier, params);
+		startAW();
+	}
+
+	private void startAW(){
```

Comment:
This definitely needs changing. Name needs to be changed, way the logic works needs to be changed.